apps/frontend/src/components/cards/controltable/ControlTable.vue

```diff
@@ -57,11 +57,21 @@
         </template>
 
         <template #id>
-          <ColumnHeader
-            text="ID"
-            :sort="sortId"
-            @input="set_sort('id', $event)"
-          />
+          <v-row class="pa-3">
+            <ColumnHeader
+              text="ID"
+              :sort="sortId"
+              @input="set_sort('id', $event)"
+            />
+            <v-tooltip bottom>
+              <template #activator="{on, attrs}">
+                <v-icon style="cursor: pointer" v-bind="attrs" x-small v-on="on"
+                  >mdi-help</v-icon
```

Comment:
Make the icon be in a circle and in-line with the text.  Use the 'Help (?)' button on the 'load from s3 bucket' page as an example.

Reply:
Or perhaps make it the "info" (the about example) button because it doesn't make the user think that the id is questionable.

Reply:
There is another example of this for the loading screen in the top right corner.